scripts/android-versions.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
```

Comment:
When you work on supporting multi-extension repositories, consider switching these scripts from Bash to Python. It will simplify maintenance.